lib/matplotlib/tests/test_collections.py

```diff
@@ -1149,3 +1149,14 @@ def test_check_masked_offsets():
 
     fig, ax = plt.subplots()
     ax.scatter(unmasked_x, masked_y)
+
+
+def test_masked_set_offsets():
```

Comment:
Can you also add a `figures_equal()` check to this? This seems like something where we want to make sure the render of the Masked components are equivalent. So you should be able to compare against a plot with only the 3 points from a normal array.

Reply:
Fair point. I'll add that!